vector/src/gplay/java/im/vector/app/nightly/NightlyProxy.kt

```diff
@@ -44,4 +53,24 @@ class NightlyProxy @Inject constructor() {
                     }
                 }
     }
+
+    private fun canDisplayPopup(): Boolean {
+        if (BuildConfig.APPLICATION_ID != "im.vector.app.nightly") return false
+        val sharedPreferences = DefaultSharedPreferences.getInstance(context)
```

Comment:
we've recently added a provider for the default shared prefs via the annotation `@DefaultPreferences`
https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/core/di/SingletonModule.kt#L217

Reply:
ah yes, I will use it, thanks!

Reply:
41d35e68a05a6c8cbbb2e006ee947b35f6f83428